next.clayui.com/content/docs/components/Panel.mdx

```diff
@@ -20,6 +20,11 @@ import {Panel} from '../../../src/components/clay/Panel';
 </div>
 </div>
 
+<div class="clay-site-alert alert alert-warning">
+	<strong class="lead">Warning</strong>
+	Before you get started with this doc, read about <a href="/docs/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
+</div>
+
```

Comment:
@diegonvs for now let's remove this from all documents and then we can add it again. Leave only the documents they already had.

Reply:
reverted!